ThirdPartyNotice.html

```diff
@@ -1,4 +1,34 @@
-<!doctype html><html lang="en"><head><title>NOTICES AND INFORMATION</title><style>pre{white-space:pre-wrap;background:#eee;padding:24px}</style></head><body><h1>NOTICES AND INFORMATION</h1><h2>Do Not Translate or Localize</h2><p>This software incorporates material from third parties. Microsoft makes certain open source code available at <a href="https://3rdpartysource.microsoft.com">https://3rdpartysource.microsoft.com</a>, or you may send a check or money order for US $5.00, including the product name, the open source component name, and version number, to:</p><address>Source Code Compliance Team<br>Microsoft Corporation<br>One Microsoft Way<br>Redmond, WA 98052<br>USA</address><p>Notwithstanding any other terms, you may reverse engineer this software to the extent required to debug changes to any libraries licensed under the GNU Lesser General Public License.</p><ol><li><details><summary>aws-sign2 0.7.0 - Apache-2.0</summary><p><a href="https://github.com/mikeal/aws-sign#readme">https://github.com/mikeal/aws-sign#readme</a></p><pre>Apache License
+<!doctype html>
```

Comment:
Don't manually edit this file. It's generated dynamically and that should only happen _after_ you merge changes to package.json/package-lock.json because it's based off of the dev ops build from master. There's instructions in the OneNote - let me know if you need help